README.md

```diff
@@ -68,7 +68,7 @@ const postGraphileMiddleware = postgraphile(databaseUrl, "app_public", {
 
 This plugin adds the following options to the PostGraphile library options:
 
-```ts
+````ts
```

Comment:
This was likely Prettier escaping a code block inside a code block; I don't think you need it so please go back to just 3 backticks
````suggestion
```ts
````